### ISSUE TITLE
Priest Mind Control ability issues

### ISSUE THREAD

**Issue by martinjlowm** [elysium-project/server#525]
There's two issues mind control issues to look at:

- Using abilities while mind controlling does not break the control. The controlled target is unable to use any abilities, because the player is currently attempting to cast. 
- Mind controlling PvP targets starts the MC AI as used in boss fights (although MC'ing warriors could potentially use all their cooldowns, Recklessness and Shield Wall etc.). This is obviously not intended!

**Comment by Cabro** [comment]
True, you can poly sheeped units, and it should be possible, just that you shouldn't have any control over the movement/attacks of the unit, and you do.
Also, the NPCs that were MCed change their status to yellow bar, even if they were hostile. Possibly this breaks the PVP units in cities and makes them targettable in tab for same faction players.

**Comment by toinetoine** [comment]
Another issue is changing zones while being mind controlled. In this [video](https://www.youtube.com/watch?v=bdyX5nWpKPU) you can see being mind controlled during the Zeppelin switching zones causes the player to fall through the world.
_[Elysium PvP server 2017-10-04]_
